docs/development/workflow/git_edit_workflow_examples.rst

```diff
@@ -52,98 +43,101 @@ Grab the latest updates to astropy
 A few steps in this tutorial take only a single command. They are broken out
 separately to outline the process in words as well as code.
 
-Inform your local copy of Astropy about the latest changes in the development
+Inform your local copy of ``astropy`` about the latest changes in the development
 version with::
 
     git fetch astropy --tags
 
 Set up an isolated workspace
 ============================
+
 + Make a new `git`_ branch for fixing this issue, check it out, and let my
   GitHub account know about this branch::
 
-    git branch fix-1761 astropy/master   # branch based on latest from GitHub
-    git checkout fix-1761                   # switch to this branch
-    git push --set-upstream origin fix-1761 # tell my github acct about it
+    git checkout astropy/master -b fix-1761 # branch based on latest from GitHub
+                                            # and switch to this branch
+    git push your-user-name fix-1761        # tell my GitHub acct about it
 
-+ Make a python environment just for this fix and switch to that environment.
-  The example below shows the necessary steps in the Anaconda python
++ Make a Python environment just for this fix and switch to that environment.
+  The example below shows the necessary steps in the Miniconda/Anaconda Python
   distribution::
 
-        conda create -n apy-1761 --clone root  # Anaconda distribution only
-        source activate apy-1761
+    conda create -n apy-1761 python=3.9 # replace 3.9 with desired version
+    conda activate apy-1761
 
   If you are using a different distribution, see :ref:`virtual_envs` for
   instructions for creating and activating a new environment.
+
 + Install our branch in this environment with::
 
     pip install -e .
```

Comment:
Later in the tutorial we ask people to run pytest, we should have them install it here. You could argue for a `[all,test]` here if you want to be able to run the full suite.
```suggestion
    pip install -e .[test]
```

Reply:
@bsipocz This is my current leading theory for the issues the new astroquery contributor ran into fyi.

Reply:
@mhvk's suggestion below to turn the pytest into a call to tox is way better. Although arguably we should have them install the test extra anyway.